walkthroughs/eks/getting-started-with-appmesh/build-appserver-v2.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Build docker image
+APPSERVER_ECR_REPO=$(jq < cfn-output.json -r '.AppServerEcrRepo');
+docker build -t $APPSERVER_ECR_REPO ./yelb-appserver-v2/
+
+# Push image to ECR repo
+$(aws ecr get-login --no-include-email)
```

Comment:
We'll also need a variant for AWS CLI v2 (see https://github.com/aws/aws-app-mesh-examples/pull/319 for reference).

Reply:
Added.